docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -174,10 +174,86 @@ on open location this_URL
 end open location 
 ```
 
-> **Tip:** To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
-> 
->     do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
-In the Script Editor, save your short script as an “Application Bundle”.
+{% include info title="Tip" content="To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
+    do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
+    In the Script Editor, save your short script as an “Application Bundle”." %}
+
+Find the saved Application Bundle, and Open Contents. Then, open the info.plist file and add the following to the plist dict:
```

Comment:
Italic or code filenames (info.plist).

Also I am not sure what "Find the saved Application Bundle, and Open Contents" means. Is it possible to clarify it?

Reply:
I can add a screenshot.

Reply:
Waiting on the latest preview to confirm things look good.